### PR TITLE
[CSR-2120] fix: passes, failed, skipped tests counting

### PR DESCRIPTION
**Pull request by miguelangaranocurrents**
## Description

> Provide a brief description of the changes in this PR. Important to list all the changes made in overall. Describe any improvements, follow up tasks or edge cases related to this PR

This PR fixes the count of skipped tests as well as the failed and passed tests, taking into account the skipped as well.

## PR Checklist

- [x] I performed manual tests or added tests that prove my fix is effective or that my feature works.
- [x] I have performed a self-review of my code.
- [x] I have annotated my PR with comments, particularly in hard-to-understand areas.
- [x] I have considered the security implications of this work.

## Release Plan

- [x] I have added `release:<service>` labels to this PR

- What is the release order of the affected services? N/A
- What infrastructure changes are requires? N/A

## Demo

> Add screenshots or videos demonstrating the solution if applicable

https://www.loom.com/share/ba8d99a52f704719b38eaa9f8f000fdc?sid=a46ad995-8c9c-4e1a-b18b-6731e6e60a75

## Manual Testing

> Describe in steps (support it with images if needed) how to try the changes of this PR. (Even the start/setup of the services needed to try it out). If it's a bug also include reproduction steps

To reproduce:
1. With an XML JUnit file with skipped test, execute the convert command.
2. It will take the skipped as passed test

To test the fix:
1. Do the first step from previous section
2. You should see in the instance file the right stats of skipped, failed and passed tests

Use this as example XML file:

```
<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="NodeJS" tests="3" failures="1" errors="0" skipped="1" time="0.006700">
  <testsuite name="/test/nodejs/specs/server/transforms/product-card.spec.js"
    timestamp="2025-01-31T18:13:17.815Z" hostname="cgahafer-Mac14,10" errors="0" tests="2"
    failures="1" skipped="1" time="0.002541">
    <testcase
      name="FinancialDataTransformer &gt; _toBullets &gt; should return an array of strings"
      time="0.000276">
      <skipped />
    </testcase>
    <testcase
      name="FinancialDataTransformer &gt; _fetchFields &gt; should return empty if category is not found"
      time="0.001210">
      <failure message="A test message" type="testCodeFailure"><![CDATA[A test message
Error [ERR_TEST_FAILURE]: A test message
at new Promise (<anonymous>)
at Array.map (<anonymous>)]]></failure>
      <failure type="AssertionFailure" message="expected 631 to be below 200">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: e1e87782-7e0d-48dc-a937-3f6e1b931edc.]]>
        <![CDATA[Collection name: Tests.]]>
        <![CDATA[Request name: Currents API.]]>
        <![CDATA[Test description: Response time is less than 200ms.]]>
        <![CDATA[Error message: expected 631 to be below 200.]]>
        <![CDATA[Stacktrace: AssertionError: expected 631 to be below 200
   at Object.eval sandbox-script.js:2:1).]]>
      </failure>
    </testcase>
    <testcase
      name="FinancialDataTransformer &gt; _fetchFields &gt; should have values for category"
      time="0.000391" />
  </testsuite>
</testsuites>
```

### CODE REVIEW

**Review by twk3 — CHANGES_REQUESTED** [review]
@miguelangaranocurrents this looks good except for one thing regarding the status

That one thing is that in all the frameworks we current support, skipped should be reported as `pending` to Currents.

I believe the jest reporter is already doing this for the testcase level, we need to do the same for the convert command.

Skipped in currents is for tests that were supposed to run, but didn't. (Sortof a cypress legacy item I think).  We count these towards failures in Currents.

File: packages/cmd/src/services/convert/utils.ts [hunk 1]
```diff
@@ -26,18 +29,20 @@ export function getTestCase(
...
     location: getTestCaseLocation(suite?.file ?? ''),
     retries: getTestRetries(failures),
-    attempts: getTestAttempts(
-      testCase,
-      failures,
-      getISODateValue(suiteTimestamp),
-      time
-    ),
+    attempts: skipped
+      ? []
```
Comment (twk3) [inline packages/cmd/src/services/convert/utils.ts:39]: @miguelangaranocurrents it looks like we need to put in a skipped attempt here (which we should add handling to getTestAttemps to do). Otherwise our dashboard UI for test details is a bit broken (and throws errors on loading stdout) If we have no attempt at all.

The attempt `status` should be `skipped` and the `_s` should be `pending`, to match what playwright ends up setting.

**Review by twk3 — CHANGES_REQUESTED** [review]

**Review by twk3 — APPROVED** [review]
Thanks @miguelangaranocurrents 

This looks to be working as expected
